src/getDuplicatedPropsDesc.ts

```diff
@@ -0,0 +1,23 @@
+export default function getDuplicatedPropsDesc(
+  story: any,
+  propTablesComponents: any
+): string {
+  const { propsDescription } = story
+  let duplicatedPropName = ''
+
+  if (propsDescription) {
+    Object.keys(propsDescription).some(propName => {
+      const isDuplicated = propTablesComponents.every(({ component }: any) => {
```

Comment:
![screenshot from 2018-07-26 11-47-00](https://user-images.githubusercontent.com/13316015/43238591-acb76790-90c9-11e8-96a1-eafff61bfcd3.png)

Seems like shows a warning even if a story has only one propsTable.
Instead of checking every components to have same prop, we should check `count > 1`.

Reply:
lol, I didn't think about that. I'll fix this. thanks!